playbooks/acs.yml

```diff
@@ -29,6 +29,14 @@
 - name: Handle secrets
   ansible.builtin.import_playbook: secrets.yml
 
+- name: Check acs edition
```

Comment:
if thinking bigger is allowed, this is the start of the replacement for the common role that we could call "dynamic facts", let's have a dedicated playbook for it with a more generic play name and import it here

you could also validate that acs_play_repository_acs_edition is either Enterprise or Community to avoid unwanted user mistakes